File: 🧰️framework/🛍️products/🦑️repo/🔨️modules/⌨️cli/📦️packages/🐹️go/🚀️bin/📦️main.go
// #region 🧲️Header

// 2026 Ueli Saluz <ueli@semio-tech.com>

// Native executable entry point for the repository CLI.

// #endregion 🧲️Header

package main

import (
	"fmt"
	"os"

	cli "github.com/usalu/semio/repo/cli"
)

// 🚀️main runs the repository CLI and reports a failure on standard error.
func main() {
	if err := cli.RunCLI(); err != nil {
		fmt.Fprintln(os.Stderr, err)
		os.Exit(1)
	}
}

File: 🧰️framework/🛍️products/🦑️repo/🔨️modules/🖥️server/🎛️coordinator/📦️packages/🦀️rust/📦️main.rs
//#region 🧲️Header

// 2026 Ueli Saluz <ueli@semio-tech.com>

// The coordinator binary: nothing but a delegation to the crate entry point.

//#endregion 🧲️Header

/// ▶️ Starts the coordinator from the ambient environment and blocks until the process is killed.
fn main() {
    if let Err(failure) = semio_framework_repo_coordinator::main_blocking() {
        eprintln!("repo coordinator: {failure}");
        std::process::exit(1);
    }
}
